packages/interface-adapter/test/fabric-evm-getId.test.ts

```diff
@@ -29,18 +34,24 @@ async function prepareGanache(
   });
 }
 
-describe("fabric-evm getId Overload", function() {
-  it("returns networkID as valid string instead of number w/ fabric-evm=true", async function() {
-    const preparedGanache = await prepareGanache(true);
-    const networkID = await preparedGanache.interfaceAdapter.getNetworkId();
-    assert(typeof networkID === "string");
-    await preparedGanache.server.close();
+describe("fabric-evm getId Overload", function () {
+  it("returns networkID as valid string instead of number w/ fabric-evm=true", async function () {
+    const preparedGanache = (await prepareGanache(true)) as any;
```

Comment:
Why is `as any` necessary here?  (And also below, of course.)  Especially because I notice you didn't use it in the other file...